content/recipes/reusability/miappe.md

```diff
@@ -0,0 +1,429 @@
+
+The selection of available checklists can be browsed at: [https://www.ebi.ac.uk/ena/browser/checklists](https://www.ebi.ac.uk/ena/browser/checklists). Currently, three plant specific checklists are available: [ERC000020](https://www.ebi.ac.uk/ena/browser/view/ERC000020), [ERC000035](https://www.ebi.ac.uk/ena/browser/view/ERC000035) and [ERC000037](https://www.ebi.ac.uk/ena/browser/view/ERC000037). 
+
+
+
+1. Return to the '**New Submission**' tab in the submission service. You may need to click the '**Restart Submission**' button at the bottom of the page.
+2. Find the '**Register samples**' radio button and click **'Next'**.
+3. You must choose an appropriate checklist of values to be provided for your sample: click '**Select checklist**' and expand the '**Other checklists**' group to browse checklists of this type.
+4. Select the appropriate checklist. For general purposes ERC000037 (**‘ENA Plant Sample Checklist’**) should be the default when submitting plant sequencing data. Now click '**Next**'.
+
+5. Submitters now have the option of including additional fields in their checklist. It is not necessary to include any additional fields but you can take this opportunity to see which fields are included by default, what requirements they have, and what else is available.
+
+    Click '**Next**'.
+
+
+6. Now you have the opportunity to construct a template for a submission of many samples: notice the phrase '**Template Basic Details**' at the top of the interface. This allows you to define values which are common to all samples in your submission. 
+
+    Since we are only submitting one sample, skip this screen by clicking '**Next**'.
+
+7. On the next screen, click the '**+ Add**' button to create your sample.
+
+8. Give a 'Unique name' to your sample:
+
+        barley_sample_1
+
+
+    **Hint:** This value must always be unique among all samples submitted by your account!
+
+9. Give your sample a title, e.g.:
+
+        Barley sample from <Your Town/Lab> greenhouse
+
+10. Write a brief description for your sample.
+11. Find the appropriate taxon for your sample: For example enter '**Hordeum vulgare**' into the box and select it when it appears. 
+
+    Further taxonomic details are automatically filled below this box. This box will only accept values which match with a species-rank entry in the [NCBI Taxonomy database](https://www.ncbi.nlm.nih.gov/taxonomy).
+
+12. Fill out the remaining details by reference to the values to the best of your knowledge. Only fields marked with a ***** are required, but it is best to fill as many fields as possible.
+
+    **Hint:** You can always find out more about what a field means and what information it accepts by hovering over the blue 'i'.
```

Comment:
````suggestion
Use a markdown directive (note: not supported by hackMD but supported by jupyter notebook)
``` {note}
You can always find out more about what a field means and what information it accepts by hovering over the blue 'i'.
```
````